reefperf/generators/parameters_generator.py

```diff
@@ -0,0 +1,37 @@
+from haikunator import Haikunator
+
+from reefperf.generators.registry import generators_registry
+
+
+class NodeNameGenerator(object):
+    @classmethod
+    def generate(cls, node_type):
+        return f"{node_type}-{Haikunator.haikunate(0)}"
```

Comment:
does this guarantee that we won't have a collision? I don't think it does. You can imagine what will happen (with low probability, but still), if two nodes will get the same name.